wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/model/impl/ActionImpl.java

```diff
@@ -31,21 +31,21 @@
  * @author Anton Korneta
  */
 @Entity(name = "Action")
-@Table(name = "action")
+@Table(name = "che_factory_action")
 public class ActionImpl implements Action {
 
     @Id
     @GeneratedValue
-    @Column(name = "entityid")
+    @Column(name = "entity_id")
     private Long entityId;
 
     @Column(name = "id")
     private String id;
 
     @ElementCollection
-    @CollectionTable(name = "action_properties", joinColumns = @JoinColumn(name = "action_entityid"))
-    @MapKeyColumn(name = "properties_key")
-    @Column(name = "properties")
+    @CollectionTable(name = "che_factory_action_properties", joinColumns = @JoinColumn(name = "action_entity_id"))
```

Comment:
Why do you use plural for che_factory_action_propertie**s** table name here but single for [che_factory_image](https://github.com/eclipse/che/pull/4192/files#diff-e520532e8b7fceec9cedf14236d578ffR89) table name?